racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -50,66 +75,82 @@ const props = defineProps({
 <template>
   <BreadCrumbs :datacenterName="props.name" />
 
-  <SearchBar
-    v-if="datacenters.length"
-    viewTitle="Datacenter Details"
-    searchedItem="datacenter"
-    :items="datacenters"
-  />
+  <div class="flex justify-end items-center p-4 px-20">
+    <input
+      type="text"
+      v-model="input"
+      class="w-96 rounded-md focus:border-purple-700 focus:outline-none border-2 border-solid bg-white py-1.5 pl-3 pr-10 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 sm:text-sm sm:leading-6"
+      :placeholder="`Filter room by name`"
+      required
+    />
+  </div>
 
-  <div v-if="datacenterDetails">
+  <div v-if="datacenterDetails" class="flex justify-center my-auto mx-auto">
     <div class="relative overflow-x-auto py-20">
-      <h2 class="text-3xl font-medium flex justify-center capitalize py-16">
-        {{ name }} Datacenter
-      </h2>
-      <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
+      <div @click="invertRoomSort()" class="flex">
+        <div v-if="alphabeticalOrder">
+          <BarsArrowDownIcon class="h-7 w-7" />
+        </div>
+        <div v-else>
+          <BarsArrowUpIcon class="h-7 w-7" />
+        </div>
+        <p>Sort rooms</p>
+      </div>
+
+      <table
+        class="min-w-[60vw] max-w-[60vw] text-base text-center text-gray-500 dark:text-gray-400 table-fixed"
+      >
         <thead
-          class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400"
+          class="text-lg text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400 h-24"
         >
           <tr>
-            <th scope="col" class="px-6 py-3">Name</th>
-            <th scope="col" class="px-6 py-3">Area (m²)</th>
-            <th scope="col" class="px-6 py-3">Number of racks</th>
-            <th scope="col">Access to the room</th>
+            <th scope="col" class="w-96">Name</th>
+            <th scope="col" class="w-96">Area (m²)</th>
+            <th scope="col" class="w-96">Number of racks</th>
+            <th class="w-0.5"></th>
           </tr>
         </thead>
 
-        <tbody v-for="room in datacenterDetails.rooms" :key="room.name">
-          <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700 text-black">
-            <th
-              scope="row"
-              class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
+        <template v-if="filteredRooms!.length > 0">
```

Comment:
You could remove the `!` operator if you verify `filteredRooms` is defined at line 88 above.